tests/api_test/api_test.vcxproj

```diff
@@ -82,7 +82,7 @@
     <LinkIncremental>false</LinkIncremental>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
-    <LinkIncremental>true</LinkIncremental>
+    <LinkIncremental>false</LinkIncremental>
```

Comment:
What do the changes in this file do?  I'm trying to match them with something in the PR description...
Same question on the other vcxproj files in this PR